packages/pwa/app/utils/url.js

```diff
@@ -103,62 +119,67 @@ export const searchUrlBuilder = (searchTerm) => `/search?q=${searchTerm}`
 
 /**
  * Returns a relative URL for a locale short code.
+ * Based on your app configuration, this function will replace your current locale shortCode with a new one
  *
  * @param {string} shortCode - The locale short code.
- * @param {Object} [opts] - Options, if there's any.
- * @param {string[]} opts.disallowParams - URL parameters to remove
+ * @param {object} [opts] - Options, if there's any.
  * @param {Object} opts.location - location object to replace the default `window.location`
  * @returns {string} - The relative URL for the specific locale.
  */
 export const getUrlWithLocale = (shortCode, opts = {}) => {
```

Comment:
```suggestion
export const getPathWithLocale = (shortCode, opts = {}) => {
```

How about renaming this to be more accurate? That it returns only the url path, similar to how `buildPathWithUrlConfig` is named.

Reply:
I've changed the name as suggested. 👍